### DISCUSSION TITLE
The new tins game in and they are B I G

### DISCUSSION THREAD

**Discussion by Wollivan** [Wollivan/SquadronLeader#36] [Announcements]
They are a lot bigger than I thought they were going to be.

It's not necessarily a bad thing, but I don't think this is the size of old or even modern tobacco tins, and much MUCH bigger than mint tins.

Here are some pictures:
![PXL_20220513_144202661.jpg](https://user-images.githubusercontent.com/91621088/168308500-e408a77c-20a2-4208-b27e-73a361174a14.jpg)
![PXL_20220513_144228831.jpg](https://user-images.githubusercontent.com/91621088/168308518-cf88ca6a-6a02-4673-a2d5-d51e6beb0b0c.jpg)
![PXL_20220513_144211686.jpg](https://user-images.githubusercontent.com/91621088/168308530-7c3dc70d-8c73-4676-8056-a569416c98d1.jpg)

**Comment by Winehorn** [comment]
More space for everything! 😁 Can you confirm that it's around 10x8x2.5cm?

**Reply by Wollivan** [reply]
I will have a look for a ruler in the morning :)
